ece_core/18614.md

```diff
@@ -0,0 +1,19 @@
+# 18-614: Microelectromechanical Systems (MEMS)
+
+## Course Overview
+
+This course covers fabrication and design fundamentals for Microelectromechanical Systems at the micro scale. Expect to learn basic principles from microstructure fabrication, mechanics of silicon and thin-film materials, electrostatic force, capacitive motion detection, fluidic damping, piezoelectricity, piezoresistivity, and thermal micromechanics. 
```

Comment:
What is the "micro scale"?

Reply:
Besides these topics, what type of skills do you expect to gain from this course?